pyanaconda/packaging/dnfpayload.py

```diff
@@ -702,9 +708,13 @@ def updateBaseRepo(self, fallback=True, checkmount=True):
                           method.method, e)
 
             try:
+                if hasattr(method, "proxy"):
+                    proxy = method.proxy
+                else:
+                    proxy = None
```

Comment:
You can use the ``getattr()`` function with ``None`` used as the default value (second argument) here.